en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -0,0 +1,62 @@
+
+1. Download **ByteDance DevTools** on [ByteDance Official Document](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download).
+
+2. Log in to [Developer Platform](https://microapp.bytedance.com/)to find appid. For details, please refer to the official details [Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game)。
+
+    ![](https://sf1-ttcdn-tos.pstatp.com/obj/website-img/f296a9f80eaeb40f4af38e8a4e25e17e_12.png)
+
+3. Select the **ByteDance Mini Game** in the **Platform** of the **Build** panel, fill in the `mini game appid`, and then click **Build**
+
+    ![](./publish-bytedance-mini-game/build.jpg)
+
+4. Preview game
+
+    Following this process, a `bytedance-mini-game` folder will be generated in the project's **build** directory (the name of the folder is based on the **Build Task Name**),then you can open your d
+
+    ![](./publish-bytedance-mini-game/tool.jpg)
+
+## Build Options
+
+Options | Optional or not | Default | Explanation
+- | - | - | -
+**appid** | Required | `testId` | The appid of the ByteDance Mini Games, it will be written to `project.config.json` file.
+**Remote server address** | Optional | Empty | The remote server address. Resources will then be obtained from this address. 
+**Open data context root** | Optional | Empty | If an Open Data Context exists, use this root to specify the relative path of the Open Data Context folder in the build directory so that the directory is not overwritten or modified during the build.
+**Orientation** | Required | `landscape` | Device orientation, it will be written to `game.json` file.
+
+##  Resource Management for ByteDance Mini Game Environment
+
+In a **ByteDance Mini Game** environment, resource management is the most special part. It differs from the browser in the following four points:
+
+1. The size of the **ByteDance Mini Game** package cannot exceed **4MB**, including all the code and resources. Additional resources must be downloaded via web request.
+
+2. For files downloaded from a remote server, the **ByteDance Mini Game** environment does not have the browser's caching and outdated update mechanism.
+
+3. For the resources in the **ByteDance Mini Game** package, they are not loaded on demand in the mini game environment, but rather all the resources in the package are loaded at once, and then the game page is launched.
+
+4. You cannot download script files from a remote server.
+
+This brings up two key issues, home page loading speed and remote resource caching and version management. For the home page loading speed, we recommend that developers only save the script file in the **ByteDance Mini Game** package, and all other resources are downloaded from the remote server. As for downloading, caching and version management of remote resources, Cocos Creator 3D has done the job for developers.
+
+Specifically, developers need to do:
```

Comment:
Specifically, developers need to do:
-->
Specifically, developers need to do the following: